spectre_analyses/tasks.py

```diff
@@ -57,6 +58,31 @@ def _load_data(dataset_name) -> ty.Dataset:
         return spdata.reader.load_txt(infile)
 
 
+class Cleanup(object):
```

Comment:
All classes since Python 3.1 already inherit from `object` – implicitly.

Reply:
It changes nothing.